exporter/opensearchexporter/sso_model.go

```diff
@@ -43,13 +43,13 @@ type ssoSpan struct {
 		SchemaURL              string         `json:"schemaUrl"`
 		Version                string         `json:"version"`
 	} `json:"instrumentationScope,omitempty"`
-	Kind         string         `json:"kind"`
-	Links        []ssoSpanLinks `json:"links,omitempty"`
-	Name         string         `json:"name"`
-	ParentSpanID string         `json:"parentSpanId"`
-	Resource     map[string]any `json:"resource,omitempty"`
-	SpanID       string         `json:"spanId"`
-	StartTime    time.Time      `json:"startTime"`
+	Kind         string            `json:"kind"`
+	Links        []ssoSpanLinks    `json:"links,omitempty"`
+	Name         string            `json:"name"`
+	ParentSpanID string            `json:"parentSpanId"`
+	Resource     map[string]string `json:"resource,omitempty"`
```

Comment:
Resource attributes can be more than just strings.  How does this handle non-string-valued resource attributes?